source/wp-content/themes/wporg-showcase-2022/functions.php

```diff
@@ -257,3 +259,57 @@ function set_site_breadcrumbs( $breadcrumbs ) {
 	$breadcrumbs[0]['title'] = __( 'Showcase', 'wporg' );
 	return $breadcrumbs;
 }
+
+/**
+ * Append an optimized site name.
+ *
+ * @param array $title {
+ *     The document title parts.
+ *
+ *     @type string $parts   Title of the viewed page.
+ *     @type string $page    Optional. Page number if paginated.
+ *     @type string $tagline Optional. Site description when on home page.
+ *     @type string $site    Optional. Site title when not on home page.
+ * }
+ * @return array Filtered title parts.
+ */
+function document_title( $parts ) {
+	global $wp_query;
+
+	if ( is_front_page() ) {
+		$parts['title']   = $parts['tagline'];
+		$parts['tagline'] = __( 'WordPress.org', 'wporg' );
+	} else {
+		if ( is_single() ) {
+			$parts['title'] = sprintf( esc_attr__( '%s - WordPress Showcase', 'wporg-showcase' ), esc_attr( $parts['title'] ) );
```

Comment:
Due to the lack of output in this function, and lack of escaping elsewhere within it, you can safely drop the esc_attr here IMHO.

```suggestion
			$parts['title'] = sprintf( __( '%s - WordPress Showcase', 'wporg-showcase' ), $parts['title'] );
```